src/Apps/Artwork/Components/ArtworkDetails/RequestConditionReport.tsx

```diff
@@ -0,0 +1,244 @@
+    setShowErrorModal(true)
+  }
+
+  function trackRequestFailure(error: string) {
+    trackEvent({
+      action_type: Schema.ActionType.RequestConditionReportFailed,
+      error_messages: [error],
+    })
+  }
+
+  function trackRequestSuccess() {
+    trackEvent({
+      action_type: Schema.ActionType.RequestConditionReportSubmitted,
+    })
+  }
+
+  const handleLoginClick = () => {
+    trackEvent({
+      action_type: AnalyticsSchema.ActionType.Click,
+      subject: AnalyticsSchema.Subject.Login,
+    })
+
+    return authentication.login(mediator)
+  }
+
+  const handleRequestConditionReportClick = () => {
+    setRequesting(true)
+
+    requestConditionReport()
+      .then(data => {
+        const theData = data as any
+        if (theData.requestConditionReport) {
+          setRequesting(false)
+          trackRequestSuccess()
+          setShowRequestedModal(true)
+        } else {
+          handleMutationError(new Error("Unknown error"))
+        }
+      })
+      .catch(error => {
+        handleMutationError(error)
+      })
+  }
+
+  const UnauthenticatedContent: React.FC = () => (
+    <>
+      <Button size="small" variant="secondaryGray" onClick={handleLoginClick}>
+        Log in
+      </Button>
+      <Sans display="inline" ml={1} size="2">
+        to request
+      </Sans>
+    </>
+  )
+
+  const AuthenticatedContent: React.FC = () => (
+    <Button
+      size="small"
+      width={180}
+      variant="secondaryGray"
+      onClick={handleRequestConditionReportClick}
+    >
+      {requesting ? <Spinner size="small" /> : "Request condition report"}
+    </Button>
+  )
+
+  const RequestedConditionReportModal: React.FC = () => (
+    <Modal
+      title="Condition report requested"
+      onClose={() => {
+        setShowRequestedModal(false)
+      }}
+      show={showRequestedModal}
+    >
+      <Serif mt={1} color="black100" textAlign="center" size="4">
+        We have received your request. The condition report will be sent to{" "}
+        {user && user.email}.
+      </Serif>
+
+      <Serif mt={2} textAlign="center" color="black100" size="4">
+        For questions, contact{" "}
+        <Link href="mailto:specialist@artsy.net">specialist@artsy.net</Link>.
+      </Serif>
+
+      <Button block mt={4} onClick={() => setShowRequestedModal(false)}>
+        OK
+      </Button>
+    </Modal>
+  )
+
+  return (
+    <>
+      <Row>
+        <Col>
+          {isLoggedIn ? <AuthenticatedContent /> : <UnauthenticatedContent />}
+        </Col>
+      </Row>
+      <RequestedConditionReportModal />
+      <ErrorModal
+        show={showErrorModal}
+        onClose={() => {
+          setShowErrorModal(false)
+        }}
+      />
+    </>
+  )
+})
+
+export const RequestConditionReportQueryRenderer: React.FC<{
+  saleArtworkID: string
+}> = ({ saleArtworkID }) => {
+  const { relayEnvironment } = useSystemContext()
+
+  return (
+    <QueryRenderer<RequestConditionReportQuery>
```

Comment:
What are the SSR implications here? Is it okay to add a `QueryRenderer` deep down the component tree? (which, in this case, is reasonable to me since we are conditionally showing the `<RequestConditionReport>` component) It might make it difficult to track what requests are being made in a single page and also could slow down the page load a bit.

cc @zephraph

Reply:
Good question! I might need to explicitly defer rendering to the client so that this isn't rendered server-side.

Reply:
Use `SystemQueryRenderer` instead of `QueryRenderer` and it will take care of deferring.

Reply:
Ahh, fantastic. Thanks @mzikherman!